lib/config/fileconf.go

```diff
@@ -1740,17 +1740,32 @@ type BPF struct {
 
 	// CgroupPath controls where cgroupv2 hierarchy is mounted.
 	CgroupPath string `yaml:"cgroup_path"`
+
+	// UDPDisableTracing disables UDP connection events.
+	UDPDisableTracing string `yaml:"udp_disable_tracing,omitempty"`
```

Comment:
Why a string?

Reply:
This allows for a wider range of values and easy distinction of empty vs false. Also follows the other Enabled field in the same config.